service/controller/cluster_framework.go

```diff
@@ -17,6 +17,7 @@ import (
 	awsclient "github.com/giantswarm/aws-operator/client/aws"
 	"github.com/giantswarm/aws-operator/service/controller/v1"
 	"github.com/giantswarm/aws-operator/service/controller/v10"
+	v10adapter "github.com/giantswarm/aws-operator/service/controller/v10/adapter"
 	v10cloudconfig "github.com/giantswarm/aws-operator/service/controller/v10/cloudconfig"
```

Comment:
General question: why we need `v10` imports here? It feels like some bug potential.

Reply:
I do not see a way around this. We import all other packages as well. Something to think about for the future on how we can make such configurations better.